src/ansys/aedt/core/modeler/circuits/primitives_maxwell_circuit.py

```diff
@@ -109,27 +109,27 @@ def create_resistor(self, name=None, value=50, location=None, angle=0, use_insta
         if location == None:
             location = []
 
-        id = self.create_component(
+        component = self.create_component(
             name,
             component_library="Passive Elements",
             component_name="Res",
             location=location,
             angle=angle,
             use_instance_id_netlist=use_instance_id_netlist,
         )
-
-        id.set_property("R", value)
-        id.set_property("Name", name)
-        return id
+        component.set_property("R", value)
```

Comment:
@DaveTwyman do not use the set_property on a property that doesn't exist, otherwise this will be created. You can access this property through parameters.